static/app/views/issueList/overview.tsx

```diff
@@ -975,18 +1056,31 @@ class IssueListOverview extends React.Component<Props, State> {
     if (itemIds.length && currentQueryCount) {
       const inInboxCount = itemIds.filter(id => GroupStore.get(id)?.inbox).length;
       currentQueryCount.count -= inInboxCount;
+      // TODO(Kelly): update once issue-list-removal-action feature is stable
+      if (!hasIssueListRemovalAction) {
+        this.setState({
+          reviewedIds: itemIds,
+          forReview: true,
+        });
+      }
       this.setState({
         queryCounts: {
           ...queryCounts,
           [query as Query]: currentQueryCount,
         },
         itemsRemoved: itemsRemoved + inInboxCount,
-        reviewedIds: itemIds,
-        forReview: true,
       });
     }
   };
 
+  onIssueAction = (itemIds: string[]) => {
+    GroupStore.remove(itemIds);
```

Comment:
i think in the future we might want to wait until the api call returns successfully